plugins/editor/src/editor/DlgAbout.cpp

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: BSD-2-Clause
+
+// This code is part of the sfizz library and is licensed under a BSD 2-clause
+// license. You should have receive a LICENSE.md file along with the code.
+// If not, contact the sfizz maintainers at https://github.com/sfztools/sfizz
+
+#include "DlgAbout.h"
+#include "GUIComponents.h"
+#include "GitBuildId.h"
+#include "NativeHelpers.h"
+
+#include "utility/vstgui_before.h"
+#include "vstgui/vstgui.h"
+#include "utility/vstgui_after.h"
+
+#define CMAKE_STR(x) _CMAKE_STR(x)
+#define _CMAKE_STR(x) #x
+
+SAboutDialog::SAboutDialog(const CRect& bounds)
+    : CViewContainer(bounds)
+{
+    SharedPointer<CBitmap> logo = owned(new CBitmap("logo_orange.png"));
+    setBackgroundColor(CColor(0x00, 0x00, 0x00, 0xc0));
+
+    CView* aboutView = nullptr;
+    {
+        auto createaboutView = [](const CRect& bounds, int, const char*, CHoriTxtAlign, int) {
+            CViewContainer* container = new CViewContainer(bounds);
+            container->setBackgroundColor(CColor(0x00, 0x00, 0x00, 0x00));
+            return container;
+        };
+
+        auto createLogo = [&logo](const CRect& bounds, int, const char*, CHoriTxtAlign, int) {
+            CViewContainer* container = new CViewContainer(bounds);
+            container->setBackgroundColor(CColor(0x00, 0x00, 0x00, 0x00));
+            container->setBackground(logo);
+            return container;
+        };
+
+        auto createInfoBox = [](const CRect& bounds, int, const char*, CHoriTxtAlign align, int fontsize) {
+            CMultiLineTextLabel* lbl = new CMultiLineTextLabel(bounds);
+            auto font = makeOwned<CFontDesc>("Roboto", fontsize);
+            lbl->setFont(font);
+            lbl->setFontColor(CColor(0xff, 0xff, 0xff, 0xff));
+            lbl->setFrameColor(CColor(0x00, 0x00, 0x00, 0x00));
+            lbl->setBackColor(CColor(0x00, 0x00, 0x00, 0x00));
+            lbl->setHoriAlign(align);
+            lbl->setText(
+                "Version " + UTF8String(CMAKE_STR(SFIZZ_VERSION)) + "." + UTF8String(GitBuildId) + "\n" +
```

Comment:
Only concatenate the ".GitBuildId" part if the GitBuildId is non-empty, otherwise we get this superflous dot character